docs/connected_orgs.rst

```diff
@@ -1,14 +1,76 @@
-Connect persistent orgs
+Connect Persistent Orgs
 =======================
+In order to connect persistent orgs such as a Developer Edition, Enterprise Edition, or Sandbox org to CumulusCI, you first need to have a Connected App configured in a persistent Salesforce org.
 
-Connect a production org
-------------------------
 
-Connect a sandbox org
----------------------
 
-Global orgs
------------
+The ``org connect`` Command
+---------------------------
+To connect to a persistent org use::
 
-Use a custom connected app
+    $ cci org connect <org_name>
+
+This will automatically open a browser window pointed to a Salesforce login page.
+The provided ``<org_name>`` will be the name that CumulusCI associates with org you log into.
+
+If you're connecting to a sandbox use the ``--sandbox`` flag.
+
+By default, ``cci org connect`` stores the OAuth credentials for connected orgs in a *project-specific* keychain.
+This means that an org connected while working in Project A's repository, will not be available while working in Project B's repository.
+If you would like to make the org available to *all* CumulusCI projects use the ``--global`` flag.
+
+.. note::
+    Connecting an org via ``cci org connect`` does *not* expose that org to ``SFDX``.
+
+
+Verify Your Connected Orgs
 --------------------------
+You can use ``cci org list`` to look for your org listed under the "Connected Org" table.
+
+The following shows a single persistent org connected to CumulusCI with the name "devhub".
+
+.. image:: images/connected_org.png
+
+
+You can use ``cci org browser`` to ensure that CumulusCI is able to login to the connected org::
+
+    $ cci org browser <org_name>
+
+
+
+CumulusCI's Connected App
+-------------------------
+CumulusCI comes with a Connected App that you can deploy to any org in your ``sfdx`` keychain.
```

Comment:
This section looks redundant with the "Using a Custom Connected App" section below, which does a better job of explaining it.

Reply:
@davisagli -  "Using a custom connected app" only mentions in passing the cci has an ootb connected app, and doesn't give steps on how to deploy it. I thought that this was a pre-requisite for even connecting persistent orgs, so wanted ensure there is a header for just deploying the built-in connected app.

Reply:
@Br4nd0R "deploying the built-in connected app" is not a thing that anyone has to do. The default connected app exists (in the SFDO Dev Hub) and CumulusCI uses it without needing any configuration. Using the `connected_app` task is ONLY for the edge cases where the user needs a custom connected app.